### PR TITLE
changes to readme files. used fern docs as template

### PR DESCRIPTION
**Pull request by danilopcarlotti**
Changed the README files to make them all similar to the fern docs template that was commented on previous PR's.

### CODE REVIEW

File: examples/extract_data/src/README.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ This example demonstrates how to use Airtop to extract data from a website using
 
 ## Setup environment
 
-Clone the repository and navigate to this example
+Install python 3.10 and pip if you haven't already.
```
Comment (aidylka) [inline examples/extract_data/src/README.md]: this is already in the prerequisites. Should list pip in the prerequisites if needed.

File: examples/extract_data/src/README.md [hunk 2]
```diff
@@ -23,10 +25,18 @@ Initiate the environment
...
+An API key is required to use this example. You can get one [here](https://portal.airtop.ai/api-keys). A sign-up is required.
 
-Input your Airtop key in the .env file.
+Once you have an API key, set it in the .env file.
+
+## Usage
 
 Change the variable `TARGET_URL` to your desired target in `extract_data_login.py`.
 
+To run the recipe:
```
Comment (aidylka) [inline examples/extract_data/src/README.md]: We don't call it recipe in the repo so lets call this `example` instead
```suggestion
To run the example:
```

File: examples/summarize/src/README.md [hunk 3]
```diff
@@ -11,7 +11,9 @@ This example demonstrates how to use Airtop to automate the summarization of a w
```
Comment (aidylka) [inline examples/summarize/src/README.md:12]: Change `## Overview` in line 1 to `# Overview` for consistency.

File: examples/summarize/src/README.md [hunk 4]
```diff
@@ -23,10 +25,18 @@ Initiate the environment
...
+Copy the `.env.example` file to .env:
+
+`cp .env.example .env`
+
+An API key is required to use this example. You can get one [here](https://portal.airtop.ai/api-keys). A sign-up is required.
 
-Input your Airtop key in the .env file.
+Once you have an API key, set it in the .env file.
+
+## Run the example
```
Comment (aidylka) [inline examples/summarize/src/README.md]: For consistency with the others
```suggestion
## Usage
```

File: examples/summarize/src/README.md [hunk 5]
```diff
@@ -23,10 +25,18 @@ Initiate the environment
...
 
 Change the variable `TARGET_URL` to your desired target in summarize.py.
 
+To run the recipe in the command line:
```
Comment (aidylka) [inline examples/summarize/src/README.md]: For consistency:
```suggestion
To run the example:
```

File: examples/summarize/src/README.md [hunk 6]
```diff
@@ -11,7 +11,9 @@ This example demonstrates how to use Airtop to automate the summarization of a w
 
 ## Setup environment
 
-Clone the repository and navigate to this example
+Install python 3.10 and pip if you haven't already.
```
Comment (aidylka) [inline examples/summarize/src/README.md]: same here.

File: examples/web_agent/README.md [hunk 7]
```diff
@@ -1,51 +1,68 @@
...
+## Prerequisites
 
 - Python 3.10
 - poetry==1.8.4
 - poetry-core==1.9.1
 
-To install, just run the following command after downloading the code
+## Setup environment
+
+Install python 3.10 and pip if you haven't already.
```
Comment (aidylka) [inline examples/web_agent/README.md]: same here.

File: examples/web_agent/README.md [hunk 8]
```diff
@@ -1,51 +1,68 @@
...
 
-## CREATE WORKER
+
+## Usage
 
 The worker.py is the core script of this agent.
 It does the following in order:
-- It starts a session. It's necessary to supply your own AIRTOP_API_KEY in the worker.py script. You can get one [here](https://portal.airtop.ai/api-keys) (sign-up required);
-- It accesses the target site, defined in TARGET_URL in the worker.py script;
+- It starts a session. It's necessary to supply your own AIRTOP_API_KEY in the worker.py script.
```
Comment (aidylka) [inline examples/web_agent/README.md]: Line 42 can be removed. This is defined in the .env file and setup on line 33 of this file so they wouldn't be adding it to the `worker.py` file.

File: examples/web_agent/README.md [hunk 9]
```diff
@@ -1,51 +1,68 @@
...
-## CREATE WORKER
+
+## Usage
 
 The worker.py is the core script of this agent.
 It does the following in order:
-- It starts a session. It's necessary to supply your own AIRTOP_API_KEY in the worker.py script. You can get one [here](https://portal.airtop.ai/api-keys) (sign-up required);
-- It accesses the target site, defined in TARGET_URL in the worker.py script;
+- It starts a session. It's necessary to supply your own AIRTOP_API_KEY in the worker.py script. 
+- It accesses the target site, defined in TARGET_URL in the `worker.py` script;
```
Comment (aidylka) [inline examples/web_agent/README.md:41]: if you are adding the ` quotes ` on line 43, we should be consistent throughout the file.

File: examples/web_agent/README.md [hunk 10]
```diff
@@ -1,51 +1,68 @@
...
 - It uses a prompt defined in the worker.py script to extract information from the site and requests login if needed;
 - If it's the first time, the db and the worker are created and the result of running the prompt on the site is exhibited;
 - If it's not the first time, then a comparison is made. The instruction for the comparison is on the file prompt_templates.py. If a comparison is made, it's inserted on the local db;
 - The result of running the prompt on the site is inserted in the db;
 
-## FETCH RESULTS
+To run the recipe:
```
Comment (aidylka) [inline examples/web_agent/README.md]:
```suggestion
To run the example:
```

**Review by aidylka — COMMENTED** [review]
Just a few comments but nice clean up!

Note that we still need to update the fern docs for [summarize](https://docs.airtop.ai/recipes/recipes/summarize-a-webpage) and [extracting data](https://docs.airtop.ai/recipes/recipes/extract-data-from-a-webpage-requiring-login) to provide TS and Python toggles and need a new fern doc for the web agent recipe. That should be a separate PR but just making sure this task wasn't confused for that one.

File: examples/extract_data/src/README.md (hunk 1 above)
Comment (danilopcarlotti) [reply examples/extract_data/src/README.md]: Done

File: examples/extract_data/src/README.md (hunk 2 above)
Comment (danilopcarlotti) [reply examples/extract_data/src/README.md]: Done

File: examples/summarize/src/README.md (hunk 3 above)
Comment (danilopcarlotti) [reply examples/summarize/src/README.md:12]: Done

File: examples/summarize/src/README.md (hunk 4 above)
Comment (danilopcarlotti) [reply examples/summarize/src/README.md]: Done

File: examples/summarize/src/README.md (hunk 5 above)
Comment (danilopcarlotti) [reply examples/summarize/src/README.md]: Done

File: examples/summarize/src/README.md (hunk 6 above)
Comment (danilopcarlotti) [reply examples/summarize/src/README.md]: Done

File: examples/web_agent/README.md (hunk 7 above)
Comment (danilopcarlotti) [reply examples/web_agent/README.md]: Done

File: examples/web_agent/README.md (hunk 8 above)
Comment (danilopcarlotti) [reply examples/web_agent/README.md]: Done

File: examples/web_agent/README.md (hunk 9 above)
Comment (danilopcarlotti) [reply examples/web_agent/README.md:41]: Ok, there's only this instance

File: examples/web_agent/README.md (hunk 10 above)
Comment (danilopcarlotti) [reply examples/web_agent/README.md]: Done

File: examples/extract_data/src/README.md (hunk 1 above)
Comment (aidylka) [reply examples/extract_data/src/README.md]: missed removing this sentence.

File: examples/web_agent/README.md (hunk 9 above)
Comment (aidylka) [reply examples/web_agent/README.md:41]: What I mean is you'll want to be consistent and update the other references to `worker.py` and `prompt_templates.py` to use the surrounding ` char. 
image
image

Comment (danilopcarlotti) [reply examples/web_agent/README.md:41]: Understood, fixed

**Review by aidylka — APPROVED** [review]
Changes LGTM! Feel free to merge! Thanks for the iterations.

**Pull request merged**